### PR TITLE
Fix EnvSort struct

### LINKED ISSUE #627: Fix EnvSort so that env variables are populated properly
Environment variables in `docker-compose` file are populated in random order resulting in failure of the `functional test` in `golang`.

I have used multiple `docker-compose` files to test this out from 
[fixtures](https://github.com/kubernetes-incubator/kompose/blob/master/script/test/fixtures/multiple-compose-files) folder.
`docker-compose` file used [1](https://github.com/kubernetes-incubator/kompose/blob/master/script/test/fixtures/multiple-compose-files/docker-k8s.yml) and [2](https://github.com/kubernetes-incubator/kompose/blob/master/script/test/fixtures/multiple-compose-files/docker-os.yml)

```bash
$ kompose convert --provider=openshift -f docker-k8s.yml -f docker-os.yml --stdout -j > a.json
$ kompose convert --provider=openshift -f docker-k8s.yml -f docker-os.yml --stdout -j > b.json
```
```
$ diff a.json b.json
```
```vim
126a127,130
>                     "name": "DB_DBID",
>                     "value": "openshift"
>                   },
>                   {
140,143d143
<                     "value": "openshift"
<                   },
<                   {
<                     "name": "DB_DBID",
```
Sorting this [struct](https://github.com/kubernetes-incubator/kompose/blob/master/pkg/kobject/kobject.go#L96)
```go
type EnvVar struct {
	Name  string
	Value string
}
```
 would fix this issue.
This issue is a blocker for #518

### PR DESCRIPTION
**Pull request by procrypt**
Fixes #627 and unblocks #518  

cc: @surajssd @cdrage

### CODE REVIEW

**Comment by surajssd** [comment]
Tests(unit tests, functional tests)

**Comment by procrypt** [comment]
Ping @kadel please review.

File: pkg/transformer/kubernetes/kubernetes.go [hunk 1]
```diff
@@ -480,14 +481,15 @@ func (k *Kubernetes) ConfigPVCVolumeSource(name string, readonly bool) *api.Volu
...
-	envs := []api.EnvVar{}
+	envs := transformer.EnvSort{}
 	for _, v := range service.Environment {
 		envs = append(envs, api.EnvVar{
 			Name:  v.Name,
 			Value: v.Value,
 		})
 	}
 
+	sort.Sort(envs)
```
Comment (kadel) [inline pkg/transformer/kubernetes/kubernetes.go]: Can you add please add a short comment explaining why it needs to be sorted?

**Comment by cdrage** [comment]
LGTM other than the nitpick from @kadel 's comment. I agree with adding a few comments.

**Comment by procrypt** [comment]
Thanks @kadel ;)

**Pull request merged**